src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
+
 export interface Figure {
+  shape: string,
+  color: string,
+  getArea(): number,
+}
+
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
 }
 
 export class Triangle implements Figure {
+  shape = Shape.triangle;
+
+  getArea(): number {
```

Comment:
getArea() can be moved into interface declaration

cause it's the method that is MANDATORY for all figures

Reply:
> getArea() can be moved into interface declaration
> 
> cause it's the method that is MANDATORY for all figures

As I see `getArea()` already declared in the interface, so you don't need to specify the return value type